samples/suit/smp_transfer/sysbuild/flash_companion.overlay

```diff
@@ -1,50 +0,0 @@
-/*
```

Comment:
Why it is safe to remove this overlay?

Reply:
Yes, the only purpose of this file was to assign a partition. Now that the flash_companion uses RAM to execute, it is not needed.